### PR TITLE
feat(facebook): add Facebook Pages integration with comprehensive management features

### PR DESCRIPTION
**Pull request by AlexanderHeffernan**
# Facebook Pages Integration

## Purpose
Add a Facebook Pages integration to Autohive that enables comprehensive management of Facebook business pages, including posts, comments, and analytics.

## Approach
This PR introduces a new Facebook integration using the Facebook Graph API v21.0. The integration is structured with modular action handlers and helper utilities to provide full-featured page management capabilities.

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Updates

:point_right: Added Facebook Pages integration with 8 actions: list pages, get/create/delete posts, get/manage/delete comments, and retrieve insights

:point_right: Implemented post scheduling with Unix timestamp and ISO 8601 support, enforcing 10-minute to 75-day scheduling window

:point_right: Added comprehensive test suite with 31 tests covering all actions, error handling, and edge cases

## Author(s) to check 
- [x] Project and all contained modules builds successfully
- [x] Self-/dev-tested
- [x] Unit/UI/Automation/Integration tests provided where applicable
- [x] Code is written to standards
- [x] Appropriate documentation written (code comments, internal docs)

### CODE REVIEW

**Comment by TheRealAgentK** [comment]
@AlexanderHeffernan Can you please add this integration to the repo-level README file?

File: facebook/facebook.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- actions/posts.py: Post CRUD operations
+- actions/comments.py: Comment management
+- actions/insights.py: Analytics
+"""
+
+from autohive_integrations_sdk import Integration
+import os
+import sys
+
+config_path = os.path.join(os.path.dirname(__file__), "config.json")
```
[the remark this answers is not in the document]
Comment (AlexanderHeffernan) [reply facebook/facebook.py]: I had a look into this, but at least on my end, `Integration.load()` without arguments doesn't find the config file. It looks for `config.json` relative to the SDK's install location (`.venv/Lib/config.json`) rather than the calling module's directory. When I tested other integrations using just `Integration.Load()`, they fail with the same `ConfigurationError`. Maybe this isn't the case when uploading integrations to Autohive, but that is what I was running into when trying to run the tests I had written locally.

The explicit path pattern is also used by github and slider, and their tests load without issues for me. Maybe I've set something up incorrectly on my machine, but it seems like the SDK would need some changes to automatically find the config file relative to where it's being called from. Happy to look into this further in the future if you'd like?

**Review by TheRealAgentK — APPROVED** [review]
Great work @AlexanderHeffernan !!!
